src/main/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcFooterItems.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.config
+
+import javax.inject.Inject
+import play.api.i18n.Messages
+import play.api.mvc.RequestHeader
+import uk.gov.hmrc.govukfrontend.views.viewmodels.footer.FooterItem
+import uk.gov.hmrc.hmrcfrontend.config.AccessibilityStatementConfig
+
+class HmrcFooterItems @Inject()(accessibilityStatementConfig: AccessibilityStatementConfig) {
+  def get(implicit messages: Messages, request: RequestHeader): Seq[FooterItem] =
+    footerItemForKey("cookies").toSeq ++
+      accessibilityLink ++
+      footerItemForKey("privacy") ++
+      footerItemForKey("termsConditions") ++
+      footerItemForKey("govukHelp")
```

Comment:
I believe this would be more efficient (and also a bit easier to read in my opinion):
```
    Seq(
      footerItemForKey("cookies"),
      accessibilityLink,
      footerItemForKey("privacy"),
      footerItemForKey("termsConditions"),
      footerItemForKey("govukHelp")
    ).flatten
```

Reply:
Sounds like a good suggestion :-)